### PR TITLE
Update "Creating repositories using the web interface"

### PR DESCRIPTION
**Pull request by bast**

### CODE REVIEW

File: creating-using-web.md [hunk 1]
```diff
@@ -25,6 +23,13 @@ learner shares their screen (video) and others can ask questions and give
 suggestions. In a larger group the instructor can demonstrate these and then
 participants can try on their own and then ask questions.
 
+
+### Note to instructors
```
Comment (rkdarst) [inline creating-using-web.md:27]: Doesn't need to be done now, but we can make a "note to instructors" directive to wrap this stuff and minimize the obtrusiveness via CSS.

Comment (bast) [reply creating-using-web.md:27]: I agree but I wanted to discuss the CSS setup anyway soon separately.

**Comment by bast** [comment]
Thanks for the tip with the screenshots but we will have to do it later. I might find time to redo them before we teach it and I will need to be a bit more careful about this in future.

**Comment by bast** [comment]
So the problem is that the screenshots have too many pixels or too few pixels?

**Comment by rkdarst** [comment]
> So the problem is that the screenshots have too many pixels or too few pixels?

Really, nothing wrong, but are a bit too many pixels wide.

Currently, it shrinks them to fit the main content column (800px) if
they are larger than that.  A full 1080p screenshot would be bad, but
these are actually OK since they are always legible.  If it is less
than 690px, there will not be any anti-aliasing and they will look
perfectly sharp (as well as being good for narrow windows during
workshops).

Since pictures to the column size by default, images too big isn't as
bad as it could be (they won't be huge and you won't have to scroll to
see them).

**Pull request merged**